kolibri/plugins/learn/assets/src/views/LibraryPage/OtherLibraries.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+
+  <div>
+    <KGrid gutter="12">
+      <KGridItem
+        :layout12="{ span: 6 }"
+        :layout8="{ span: 4 }"
+        :layout4="{ span: 4 }"
+      >
+        <h1>
+          {{ injectedtr('otherLibraries') }}
+        </h1>
+      </KGridItem>
+      <KGridItem
+        :layout12="{ span: 6 }"
+        :layout8="{ span: 4 }"
+        :layout4="{ span: 4 }"
+      >
+        <div class="sync-status">
+          <span
+            v-show="searchingOtherLibraries"
+            data-test="searching"
+          >
+            <span data-test="searching-label">{{ injectedtr('searchingOtherLibrary') }}</span>
+                &nbsp;&nbsp;
+            <span>
+              <KCircularLoader
+                type="indeterminate"
+                :stroke="6"
+              />
+            </span>
+          </span>
+          <span
+            v-show="!searchingOtherLibraries && devicesWithChannelsExist"
+            data-test="showing-all"
+          >
+            <span>
+              <KIcon
+                v-if="windowIsSmall"
+                icon="wifi"
+                class="wifi-svg"
+              />
+            </span>
+                &nbsp;&nbsp;
+            <span data-test="showing-all-label">{{ injectedtr('showingAllLibraries') }}</span>
+                &nbsp;&nbsp;
+            <span>
+              <KIcon
+                v-if="!windowIsSmall"
+                icon="wifi"
+                class="wifi-svg"
+              />
+            </span>
+          </span>
+          <span
+            v-show="!searchingOtherLibraries && !devicesWithChannelsExist"
+            data-test="no-other"
+          >
+            <span>
+              <KIcon icon="disconnected" />
+            </span>
+                &nbsp;&nbsp;
+            <span data-test="no-other-label">{{ injectedtr('noOtherLibraries') }}</span>
+          </span>
+        </div>
+      </KGridItem>
+    </KGrid>
+
+    <h2
+      v-if="pinnedDevicesExist && unpinnedDevicesExist"
+      data-test="pinned-label"
+    >
+      {{ injectedtr('pinned') }}
+    </h2>
+    <PinnedNetworkResources
+      v-if="pinnedDevicesExist"
+      data-test="pinned-resources"
+      :devices="pinnedDevices"
+      :deviceChannelsMap="deviceChannelsMap"
+      :channelsToDisplay="cardsPerRow * 2 - 1"
```

Comment:
It feels very abstract what `cardsPerRow * 2 - 1` is doing. It might be a better idea to implement a computed property with a good descriptive name/comment instead?

Reply:
Can do!

Reply:
Updated!